plugins/outputs/exec/exec.go

```diff
@@ -0,0 +1,125 @@
+package exec
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"os/exec"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/plugins/outputs"
+	"github.com/influxdata/telegraf/plugins/serializers"
+	shellquote "github.com/kballard/go-shellquote"
+)
+
+var sampleConfig = `
+  ## Command
+  command = "/usr/bin/mycollector --foo=bar"
+
+  ## Timeout for each command to complete.
+  timeout = "5s"
+
+  ## Data format to output.
+  ## Each data format has it's own unique set of configuration options, read
+  ## more about them here:
+  ## https://github.com/influxdata/telegraf/blob/master/docs/DATA_FORMATS_OUTPUT.md
+  data_format = "influx"
+`
+
+type Exec struct {
+	Command string
```

Comment:
I still think this should be an array:
```
command = ["echo", "howdy"]
```

This should be less problematic than using shellquote.Split, especially for those on Windows.